tutor/templates/config.yml

```diff
@@ -51,6 +51,7 @@ LANGUAGE_CODE: "en"
 MONGODB_HOST: "mongodb"
 MONGODB_DATABASE: "openedx"
 MONGODB_PORT: 27017
+MONGOID_AUTH_MECH: ":scram"
```

Comment:
In cs_comments_service we have the following line:

    config/mongoid.yml:    auth_mech: <%= ENV['MONGOID_AUTH_MECH'].nil? ? ':scram' : ENV['MONGOID_AUTH_MECH'] %>

Thus, `MONGOID_AUTH_MECH: ""` would be a better default value, wouldn't it?